src/full/Agda/TypeChecking/Primitive/Cubical.hs

```diff
@@ -973,15 +989,22 @@ transpSysTel' flag delta us phi args = do
       s <- reduce $ getSort t
       -- Γ ⊢ b : t[1]    Γ, i ⊢ bf : t[i]
       (b,bf) <- runNamesT [] $ do
+        let b' = Abs "i" $ unDom t
         l <- case s of
-               SSet _ -> return Nothing
-               IntervalUniv -> return Nothing
-               SizeUniv     -> return Nothing
-               LockUniv     -> return Nothing
-               Inf _ _ -> return Nothing
-               Type l -> Just <$> open (lam_i (Level l))
-               _ -> noTranspError (Abs "i" (unDom t))
-        t <- open $ Abs "i" (unDom t)
+          Univ UType l -> Just <$> open (lam_i (Level l))
+          Univ _ _     -> pure Nothing
```

Comment:
This change of the behavior for `Prop` (from exception to `Nothing`) makes the failing test for #6060 pass.
However the test passes when we have `Propω` instead of `Prop`, see #7535.

Shouldn't `Prop` be treated like `SSet` here?  If not, why not?
I think this case distinction deserves a comment on what the intended behavior is on a higher level, in particular, when to return `Nothing` and when to throw an exception.
It also feels this should not be an adhoc pattern matching, but a function to be called with a proper documentation.  There are similar matches allover the Cubical Agda code base.

Reply:
I am now distinguishing `UProp` for both `Univ` and `Inf`, erroring out rather than returning `Nothing`.
This fixes the regressions for #6060 and #7535.